algorithms/Centrality/article_rank/tg_article_rank.gsql

```diff
@@ -0,0 +1,58 @@
+CREATE QUERY tg_article_rank (STRING v_type, STRING e_type,
+ FLOAT max_change=0.001, INT max_iter=25, FLOAT damping=0.85, INT top_k = 100,
+ BOOL print_accum = TRUE, STRING result_attr =  "", STRING file_path = "") {
+/*
+ Compute the article rank score for each vertex in the GRAPH
+ In each iteration, compute a score for each vertex:
+     score = (1-damping) + damping*average outdegree*sum(received scores FROM its neighbors/average outdegree+Outdegree).
+ The article Rank algorithm stops when either of the following is true:
+ a) it reaches max_iter iterations;
+ b) the max score change for any vertex compared to the last iteration <= max_change.
+ v_type: vertex types to traverse          print_accum: print JSON output
+ e_type: edge types to traverse            result_attr: INT attr to store results to
+ max_iter; max #iterations                 file_path: file to write CSV output to
+ top_k: #top scores to output              display_edges: output edges for visualization
+ max_change: max allowed change between iterations to achieve convergence
+ damping: importance of traversal vs. random teleport
+
+ This query supports only taking in a single edge for the time being (8/13/2020).
+*/
+  TYPEDEF TUPLE<VERTEX Vertex_ID, FLOAT score> Vertex_Score;
+  HeapAccum<Vertex_Score>(top_k, score DESC) @@topScores;
```

Comment:
In accordance with the style guidelines, let's make sure all the identifiers are snake_style except for type names which are Snake_Camel.

Reply:
This goes for all the new algorithms!